unstructured_inference/models/yolox_model.py

```diff
@@ -0,0 +1,130 @@
+import os
```

Comment:
Can we rename this `yolox.py` for consistency with the other files in this directory?